composer.json

```diff
@@ -23,6 +23,8 @@
         "miljar/php-exif": "^0.6.4",
         "nelmio/cors-bundle": "^1.5",
         "nesbot/carbon": "^1.34",
+        "php-http/httplug-pack": "^1.1",
```

Comment:
should be in require-dev

Reply:
In that case the php-translation/loco-adapter needs to go into `dev` too, and I'll need to make sure bundles.php and .yml files don't break on 'non dev'. Will do.